cameo/api/designer.py

```diff
@@ -74,9 +74,11 @@ def __call__(self, strategy):
 class _OptGeneRunner(_OptimizationRunner):
     def __call__(self, strategy):
         max_evaluations = 15000
```

Comment:
I agree, but it needs a general configuration strategy with default values that can be easily defined without using 50 keyword arguments.